From 63d004a5992623cf69411a32e016a4b149d5f50c Mon Sep 17 00:00:00 2001
From: filiprrs <filiprrs@gmail.com>
Date: Fri, 8 Dec 2017 14:28:28 +0100
Subject: [PATCH 030/249] wiki: update t0lte info

* t0ltekor is now unified with t0lte so redirect
* add correct frequency bands
* update maintainers

Change-Id: I1a1fc7707a06197b7316db888060dc5a38a63f20
---
 _data/devices/t0lte.yml    | 10 ++++----
 _data/devices/t0ltekor.yml | 52 --------------------------------------
 pages/build/t0lte.md       |  5 +++-
 pages/build/t0ltekor.md    | 10 --------
 pages/info/t0lte.md        |  5 +++-
 pages/info/t0ltekor.md     | 11 --------
 pages/install/t0lte.md     |  5 +++-
 pages/install/t0ltekor.md  | 10 --------
 8 files changed, 17 insertions(+), 91 deletions(-)
 delete mode 100644 _data/devices/t0ltekor.yml
 delete mode 100644 pages/build/t0ltekor.md
 delete mode 100644 pages/info/t0ltekor.md
 delete mode 100644 pages/install/t0ltekor.md

diff --git a/_data/devices/t0lte.yml b/_data/devices/t0lte.yml
index 1326b74..9d538f6 100644
--- a/_data/devices/t0lte.yml
+++ b/_data/devices/t0lte.yml
@@ -19,13 +19,13 @@ height: 151.10 mm (5.95 in)
 image: t0lte.png
 install_method: heimdall
 kernel: android_kernel_samsung_smdk4412
-maintainers: [PoisonNinja]
-models: [GT-N7105, SGH-I317, SGH-I317M, SGH-T889]
+maintainers: [filiprrs]
+models: [GT-N7105/N7105T, SGH-I317/I317M, SGH-T889/T889V, SHV-E250S/E250K/E250L]
 name: Galaxy Note 2 (LTE)
 network:
-- {tech: 2G, bands: '850 900 1800 1900 MHz GSM' }
-- {tech: 3G, bands: '850 900 2100 MHz HSDPA' }
-- {tech: 4G, bands: '800 900 1800 2600 MHz LTE' }
+- {tech: 2G, bands: '850 900 1800 1900 MHz GSM (All models), 1800 Mhz CDMA (E250L)' }
+- {tech: 3G, bands: '850 900 1900 2100 MHz HSDPA (N7105/T), 850 1900 2100 MHz HSDPA (i317/M,T889/V), 1900 2100 Mhz HSDPA (E250S/K/L)' }
+- {tech: 4G, bands: '800 1800 2600 MHz (N7105), 1800 2600 MHz (N7105T), 700 850 1900 (i317/M), 700 (T889/V), 850 1800 Mhz (E250S), 900 1800 MHz (E250K), 850 2100 MHz (E250L)' }
 peripherals: [Accelerometer, Barometer, Compass, GPS, Gyroscope, NFC, Proximity sensor, RGB ambient light]
 ram: 2 GB
 recovery_boot: With the device powered off, hold <kbd>Home</kbd> + <kbd>Volume Up</kbd> + <kbd>Power</kbd>.
diff --git a/_data/devices/t0ltekor.yml b/_data/devices/t0ltekor.yml
deleted file mode 100644
index cc7057d..0000000
--- a/_data/devices/t0ltekor.yml
+++ /dev/null
@@ -1,52 +0,0 @@
-architecture: arm
-battery: Removable Li-Ion 3100 mAh
-bluetooth: '4.0'
-cameras:
-- {flash: 'LED', info: '8 MP'}
-- {flash: '', info: '1.9 MP'}
-carrier:
-channels: [weekly]
-codename: t0ltekor
-cpu: Cortex A9
-cpu_cores: '4'
-cpu_freq: 1.6 GHz
-current_branch: 14.1
-depth: 9.40 mm (0.37 in)
-download_boot: With the device powered off, hold <kbd>Home</kbd> + <kbd>Volume Down</kbd> + <kbd>Power</kbd>.
-gpu: Mali-400MP4
-has_recovery_partition: true
-height: 151.10 mm (5.95 in)
-image: t0lte.png
-install_method: heimdall
-kernel: android_kernel_samsung_smdk4412
-maintainers: [filiprrs, PoisonNinja]
-models: [SHV-E250S, SHV-E250K, SHV-E250L]
-name: Galaxy Note 2 (Korea)
-network:
-- {tech: 2G, bands: '850 900 1800 1900 MHz GSM (All models), 1800 MHz CDMA (E250L)' }
-- {tech: 3G, bands: '1900 2100 MHz HSDPA (All models)' }
-- {tech: 4G, bands: '900 1800 MHz (E250K), 850 2100 MHz (E250L), 850 1800 Mhz (E250S)' }
-peripherals: [Accelerometer, Barometer, Compass, GPS, Gyroscope, NFC, Proximity sensor, RGB ambient light]
-ram: 2 GB
-recovery_boot: With the device powered off, hold <kbd>Home</kbd> + <kbd>Volume Up</kbd> + <kbd>Power</kbd>.
-release: September 2012
-screen: 140 mm (5.5 in)
-screen_ppi: 265
-screen_res: 1280x720
-screen_tech: HD Super AMOLED
-sdcard: Up to 64 GB
-soc: Samsung Exynos 4412
-storage: 16/32/64 GB
-tree: android_device_samsung_t0ltekor
-type: phone
-vendor: Samsung
-vendor_short: samsung
-versions: [14.1]
-width: 80.50 mm (3.17 in)
-wifi: 802.11 a/b/g/n
-note_show: true
-note_title: TWRP recovery
-note_content: TWRP recoveries for Korean Note 2 and N7105 are now unified. Install the latest TWRP for t0lte prior to flashing.
-note_link: Link to recovery page
-note_url: https://eu.dl.twrp.me/t0lte/
-
diff --git a/pages/build/t0lte.md b/pages/build/t0lte.md
index ecca51f..5686065 100644
--- a/pages/build/t0lte.md
+++ b/pages/build/t0lte.md
@@ -3,7 +3,10 @@ sidebar: home_sidebar
 title: Build for t0lte
 folder: info
 # name of the page (/{{permalink}}.html)
-redirect_from: t0lte_build.html
+redirect_from:
+  - t0lte_build.html
+  - t0ltekor_build.html
+  - /devices/t0ltekor/build
 permalink: /devices/t0lte/build
 device: t0lte
 ---
diff --git a/pages/build/t0ltekor.md b/pages/build/t0ltekor.md
deleted file mode 100644
index be8fb45..0000000
--- a/pages/build/t0ltekor.md
+++ /dev/null
@@ -1,10 +0,0 @@
----
-sidebar: home_sidebar
-title: Build for t0ltekor
-folder: info
-# name of the page (/{{permalink}}.html)
-redirect_from: t0ltekor_build.html
-permalink: /devices/t0ltekor/build
-device: t0ltekor
----
-{% include templates/device_build.md %}
diff --git a/pages/info/t0lte.md b/pages/info/t0lte.md
index 2bf346a..dcea9f8 100644
--- a/pages/info/t0lte.md
+++ b/pages/info/t0lte.md
@@ -4,7 +4,10 @@ title: Info about t0lte
 folder: info
 layout: deviceinfo
 # name of the page (/{{permalink}}.html)
-redirect_from: t0lte_info.html
+redirect_from:
+  - t0lte_info.html
+  - t0ltekor_info.html
+  - /devices/t0ltekor/
 permalink: /devices/t0lte/
 device: t0lte
 ---
diff --git a/pages/info/t0ltekor.md b/pages/info/t0ltekor.md
deleted file mode 100644
index 432bcb2..0000000
--- a/pages/info/t0ltekor.md
+++ /dev/null
@@ -1,11 +0,0 @@
----
-sidebar: home_sidebar
-title: Info about t0ltekor
-folder: info
-layout: deviceinfo
-# name of the page (/{{permalink}}.html)
-redirect_from: t0ltekor_info.html
-permalink: /devices/t0ltekor/
-device: t0ltekor
----
-{% include templates/device_info.md %}
diff --git a/pages/install/t0lte.md b/pages/install/t0lte.md
index eae9ec5..35e12a4 100644
--- a/pages/install/t0lte.md
+++ b/pages/install/t0lte.md
@@ -3,7 +3,10 @@ sidebar: home_sidebar
 title: Install LineageOS on t0lte
 folder: info
 # name of the page (/{{permalink}}.html)
-redirect_from: t0lte_install.html
+redirect_from:
+  - t0lte_install.html
+  - t0ltekor_install.html
+  - /devices/t0ltekor/install
 permalink: /devices/t0lte/install
 device: t0lte
 ---
diff --git a/pages/install/t0ltekor.md b/pages/install/t0ltekor.md
deleted file mode 100644
index 5cf4f8e..0000000
--- a/pages/install/t0ltekor.md
+++ /dev/null
@@ -1,10 +0,0 @@
----
-sidebar: home_sidebar
-title: Install LineageOS on t0ltekor
-folder: info
-# name of the page (/{{permalink}}.html)
-redirect_from: t0ltekor_install.html
-permalink: /devices/t0ltekor/install
-device: t0ltekor
----
-{% include templates/device_install.md %}
-- 
2.17.1

